src/contracts/protocol/libraries/helpers/Errors.sol

```diff
@@ -104,4 +104,5 @@ library Errors {
   string public constant RESERVE_NOT_IN_DEFICIT = '102'; // The reserve is not in deficit
   string public constant MUST_NOT_LEAVE_DUST = '103'; // Below a certain threshold liquidators need to take the full position
   string public constant USER_CANNOT_HAVE_DEBT = '104'; // Thrown when a user tries to interact with a method that requires a position without debt
+  string public constant CALLER_NOT_ATOKEN_TRANSFER_ADMIN = '105'; // The caller of the function is not the RWA force transfer admin
```

Comment:
```suggestion
  string public constant CALLER_NOT_ATOKEN_TRANSFER_ADMIN = '300'; // The caller of the function is not the RWA force transfer admin
```
should probably use a separate namespace for rwa specific errors to keep backwards compatibility (if needed at any point) with upstream less painful, hence we can start these errors from 300 for ex